RHEL6_7/selinux/RestoreConfig/postupgrade.d/restoreConfig.sh

```diff
@@ -2,7 +2,27 @@
 
 SEMANAGE_EXPORT_FILE=semanage_export
 
+semanage_syntax_update() {
+
+    ftype_opts=( '-f' '-ftype')
+    bool_opts=( '-1' '-0' '--on' '--off' )
+
+    for opt in ${bool_opts[@]}
+    do
+        sed -i -r 's/'"$opt"'/-m '"$opt"'/g' "$1"
+    done
+
+    for opt in ${ftype_opts[@]}
+    do
+        sed -i -r 's/'"$ftype_opt"'\sdirectory/'"$ftype_opt"' d/g' "$1"
+    done
+}
+
+
 if test -s "$SEMANAGE_EXPORT_FILE"; then
+    if grep -q "release 7\..*" /etc/redhat-release;then
```

Comment:
This is reduntant condition as the script is processed always after the upgrade. It doesn't make sense to check it.